ci-operator/config/jws-qe/interop-ocp-ci/jws-qe-interop-ocp-ci-main__ocp-4.15-lp-interop.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: jws-interop-aws
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
```

Comment:
We should add success_rules for these configs as well as @smatula has done here
https://github.com/openshift/release/pull/48905/files#diff-15d11202448f6bccf6c53d5a172349b601451c4c1fed92a96632c94e579f936bR50